### DISCUSSION TITLE
采集延迟指标

### DISCUSSION THREAD

**Discussion by wangpeix** [alibaba/ilogtail#804] [Help] [answered]
请教下，怎么查看一个任务是否有采集延迟? 采集延迟的指标是如何计算的？

**Comment by messixukejia** [comment, marked as answer]
https://ilogtail.gitbook.io/ilogtail-docs/v/pre-release/configuration/logging

![image](https://user-images.githubusercontent.com/13250519/233018840-8c92dd3e-b531-44e4-b9c6-0d86ab31a916.png)
可以看下这两个日志，有文件offset、size之类日志

今年也在做可观测增强https://github.com/alibaba/ilogtail/discussions/716 ，会把指标系统重新设计下，需求可以在这里交流下，5月应该可以具备。
